src/script.js

```diff
@@ -470,7 +458,7 @@ function calculateExpenses() {
     }
 
 
-    
+
     // Izračunaj stroške omrežnine (glede na izbrano uporabniško skupino) - stroški novega načina omrežnine
```

Comment:
if you would use diffrent structure of price_e_cb and  cost_e_cb something like objcets then you could avoid the loops